### PR TITLE
[Review 1st] Add meta refresh tag stripping

### PR DESCRIPTION
**Pull request by djones**

### CODE REVIEW

File: helpers_test.go [hunk 1]
```diff
@@ -118,3 +118,61 @@ func TestStripSRI(t *testing.T) {
 		So(string(stripSRI([]byte(html))), ShouldEqual, expectedHtml)
 	})
 }
+
+func TestStripMetaRefresh(t *testing.T) {
+	Convey("stripMetaRefresh strips meta refresh tags", t, func() {
+		html := `
+			<html>
+			  <head>
+			    <meta http-equiv="refresh" content="30">
```
Comment (fotinakis) [inline helpers_test.go]: My only thought here is that this doesn't technically test that inner elements are NOT being stripped — ie you could have a bug where you're removing from the beginning `<meta` to the end `>` and we wouldn't know it here. Could potentially add a `<div>test inner tag</div>` or something here.

Comment (djones) [reply helpers_test.go]: @fotinakis so have a test tag like this?

```html
<div><meta http-equiv="refresh" content="30"></div>
```

Comment (fotinakis) [reply helpers_test.go]: @djones I mean more like:

```html
<meta http-equiv="refresh" content="30">
<div></div>
<meta http-equiv="refresh" content="30">
```

Comment (djones) [reply helpers_test.go]: I added a test case for this one here 5128b0e89566c2d2ecae59266b809763cd0b6772

**Pull request merged**